experimental/torch_xla2/torch_xla2/ops/jaten.py

```diff
@@ -2391,6 +2391,14 @@ def _aten_hypot(input, other):
   return jnp.hypot(input, other)
 
 
+@op(torch.ops.aten.digamma)
+def _aten_digamma(input, *, out=None):
+  res = jax.scipy.special.digamma(input).astype(jnp.float32)
+  zero_indices = jnp.argwhere(jnp.equal(input, jnp.zeros(input.shape)))
+  for idx in zero_indices:
```

Comment:
you can do
```
res = jnp.where(jnp.equal(input, jnp.zeros(input.shape), -jnp.inf, res)
```

Reply:
Thank you for the suggestion! I have done it.